packages/synapse-react-client/src/components/IconSvg/IconSvg.tsx

```diff
@@ -245,253 +249,256 @@ export type IconSvgProps = {
     case 'replyTwoTone':
-      return <ReplyTwoTone sx={sx}></ReplyTwoTone>
+      return <ReplyTwoTone {...otherProps} />
     case 'chatTwoTone':
-      return <ChatTwoTone sx={sx}></ChatTwoTone>
+      return <ChatTwoTone {...otherProps} />
     case 'accessManagement':
-      return <AccessManagement sx={sx}></AccessManagement>
+      return <AccessManagement {...otherProps} />
     case 'chevronRight':
-      return <ChevronRight sx={sx} />
+      return <ChevronRight {...otherProps} />
     case 'chevronLeft':
-      return <ChevronLeft sx={sx} />
+      return <ChevronLeft {...otherProps} />
     case 'pushpin':
-      return <PushPinTwoTone sx={sx} />
+      return <PushPinTwoTone {...otherProps} />
     case 'addBoxOutline':
-      return <AddBoxOutlined sx={sx} />
+      return <AddBoxOutlined {...otherProps} />
     case 'minusBoxOutline':
-      return <IndeterminateCheckBoxOutlined sx={sx} />
+      return <IndeterminateCheckBoxOutlined {...otherProps} />
     case 'italic':
-      return <FormatItalic sx={sx} />
+      return <FormatItalic {...otherProps} />
     case 'bold':
-      return <FormatBold sx={sx} />
+      return <FormatBold {...otherProps} />
     case 'title':
-      return <Title sx={sx} />
+      return <Title {...otherProps} />
     case 'visibility':
-      return <VisibilityTwoTone sx={sx} />
+      return <VisibilityTwoTone {...otherProps} />
     case 'visibilityOff':
-      return <VisibilityOffTwoTone sx={sx} />
+      return <VisibilityOffTwoTone {...otherProps} />
     case 'strikethrough':
-      return <StrikethroughS sx={sx} />
+      return <StrikethroughS {...otherProps} />
     case 'latex':
       return <span>TeX</span>
     case 'image':
-      return <Image sx={sx} />
+      return <Image {...otherProps} />
     case 'superscript':
-      return <Superscript sx={sx} />
+      return <Superscript {...otherProps} />
     case 'subscript':
-      return <Subscript sx={sx} />
+      return <Subscript {...otherProps} />
     case 'edit':
-      return <EditTwoTone sx={sx} />
+      return <EditTwoTone {...otherProps} />
     case 'tag':
-      return <AlternateEmail sx={sx} />
+      return <AlternateEmail {...otherProps} />
     case 'restore':
-      return <RestoreFromTrashTwoTone sx={sx} />
+      return <RestoreFromTrashTwoTone {...otherProps} />
     case 'label':
-      return <LabelTwoTone sx={sx} />
+      return <LabelTwoTone {...otherProps} />
     case 'upload':
-      return <UploadTwoTone sx={sx} />
+      return <UploadTwoTone {...otherProps} />
     case 'flag':
-      return <FlagTwoTone sx={sx} />
+      return <FlagTwoTone {...otherProps} />
     case 'email':
-      return <MailOutlineTwoTone sx={sx} />
+      return <MailOutlineTwoTone {...otherProps} />
+    case 'sortUp':
+      return <SortUp {...otherProps} />
+    case 'sortDown':
+      return <SortDown {...otherProps} />
     case 'google24':
       return <GoogleIcon24 sx={sx} />
```

Comment:
Pass {...otherProps} to this component just like all of the other icons. It uses MUI SvgIcon so it can use these props

```suggestion
      return <GoogleIcon24 {...otherProps} />
```